src/components/UI/Display/MAPI/workernodes/WorkerNodesNodePoolActions.tsx

```diff
@@ -0,0 +1,87 @@
+import { Keyboard } from 'grommet';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+import DropdownMenu, {
+  DropdownTrigger,
+  Link,
+  List,
+} from 'UI/Controls/DropdownMenu';
+
+const StyledDropdownTrigger = styled(DropdownTrigger)`
+  border-radius: ${({ theme }) => theme.rounding}px;
+`;
+
+interface IWorkerNodesNodePoolActionsProps
+  extends React.ComponentPropsWithoutRef<'div'> {
+  onRenameClick?: () => void;
+}
+
+const WorkerNodesNodePoolActions: React.FC<IWorkerNodesNodePoolActionsProps> = ({
```

Comment:
Unfortunately you cannot navigate between dropdown options with the arrow keys. This is a current limitation in our `DropdownMenu` component. Not fixing this now, since we'll replace the dropdown stuff with `grommet` dropdowns